bismuthcore/bismuthcore/bismuthconfig.py

```diff
@@ -2,12 +2,11 @@
 BismuthConfig class, derived from legacy config.py
 """
 
-import logging
 from os import path
 from pprint import pprint
 from sys import exc_info
 
-from bismuthcore.helpers import BismuthBase
+from bismuthcore.bismuthcore.helpers import BismuthBase
```

Comment:
The top level bismuthcore dir of the repo is an extra dir, not part of the library.
So there is no bismuthcore.bismuthcore in the module and this fix, as other similar imports in other files, adding an extraneous bismuthcore, would break anything requiring bismuthcore.
The pypi config therefore may be wrong as well. 
If an IDE suggested that, then its config or lib paths are wrong.

Reply:
Something is odd with the setup.py, the only suggestion I received from my IDE was that imports could not resolve. Pytest confirmed that fact. I'll address in `setup.py`.